docs/contributing.md

```diff
@@ -0,0 +1,122 @@
+# Contributing
+
+Contributions to Student Robitics' Runbook should follow Student Robitics'
+[Code of Conduct][code-of conduct] and made under a compatible [license][license].
+
+You may also find the [Zen of Python][pep-20] provides useful guidance.
+
+[code-of conduct]: https://opsmanual.studentrobotics.org/about-the-charity/code-of-conduct
+[license]: ./license.md
+[pep-20]: https://www.python.org/dev/peps/pep-0020/
+
+## Making big changes
+
+As the runbook is a resource which expects to be read by a large number of
+people, its current shape & content form a public API of sorts -- the way to
+navigate it as well as the locations of much of the content are likely
+muscle-memory to many contributors and readers. Making big changes to it
+therefore needs careful consideration to ensure that their workflows are not
+disrupted unduly.
+
+It's a good idea therefore to raise any large changes as a [new issue][new-issue]
+before commiting to them.
+
+Alternatively, if your change is more easily explained as a demonstration of the
+outcome, it's fine to raise a pull request showing that. Note however that your
+suggestion will still be discsused and may not be accepted, so you may wish to
+focus on a small example of the change first.
+
+[new-issue]: https://github.com/srobo/runbook/issues/new
+
+## Work in progress
+
+The runbook is very much a work in progress, in two distinct ways.
+
+Firstly: the content here is never going to be "complete". There will always be
+more that could be added or better ways to explain things. We prefer to have
+content which is present and correct, even if imperfect, where that allows for
+easier and faster contribution. This isn't to say we should be sloppy, but
+rather that we shouldn't let perfect become the enemy of good.
+
+Secondly: in a more practical sense a lot of the content here has been imported
+from legacy documentation which had a very different approach. As a result,
+there are a number of apparent conventions here that are not compatible with the
+approach described in this contribution guide. In general you are encouraged to
+follow the guidance in this document, even when working around existing pages
+which do not.
+
+## Structure
+
+The runbook aims to be available to and welcome contributions from all
+volunteers, regardless of their situation. As a result, the runbook is organised
+into topics which relate to volunteers as a whole, rather than any specific
+group.
+
+This is a deliberate change from a previous approach where content was arranged
+around the people who worked on specific areas. Unfortunately that approach lead
+to isolated silos of information which varied wildly in their internal
+structure, content approach, utility and discoverablity. While the runbook was
+pulled together from a number of those sources, it
+
```

Comment:
it

Reply:
oops, though I'd sorted that before pushing :( 41aa90b927cd4d7d109a3b5ebe4f1a53f6d87463 finishes the sentence, though I'm still not entirely happy with the wording.